dev.Dockerfile

```diff
@@ -17,7 +17,8 @@ USER node
 # Expose the port that the application listens on.
 EXPOSE 8443
 
-CMD npm install && \
+# Not sure why dev dependencies aren't automatically installed here
```

Comment:
i think technically `esbuild` is also meant to be a dev dependency but i got lazy and just made it a regular dependency. idk when is the right time to use dev vs non-dev dependencies

Reply:
oh i see, so you would have one stage in development mode for build and one stage in production mode for running your application